README.md

```diff
@@ -0,0 +1,31 @@
+### Using sequences from NCBI GenBank to estimate how many species there are in the fungi genus *Stereum*
+Sarah DeLong-Duhon
+
+Reference: Fryssouli, V., Zervakis, G., Polemis, E., & Typas, M. A. (2020). A global meta-analysis of ITS rDNA sequences from material belonging to the genus Ganoderma (Basidiomycota, Polyporales) including new data from selected taxa. MycoKeys, 75(December), 71–143. https://doi.org/10.3897/mycokeys.75.59872 
+##### Intro
+There are estimated to be 2.2 to 2.8 million fungi species on earth, but we have discovered only a small fraction of them (Hawksworth & Lücking 2017, Vu et al. 2014). Even large mushroom-forming fungi, which are easy to find and document, are not very well characterized. In Fryssouli et al. 2020, the authors used almost four thousand internal transcribed spacer (ITS) sequences from public genetic databases such as NCBI GenBank and UNITE to estimate species diversity in a genus of large shelf mushrooms, some of which have a long history of medicinal use. They found 80 *Ganoderma* species, 21 of which are not named or properly/fully identified, separated into three major lineages. 	
+
+They acquired these sequences by retrieving those labeled as genus *Ganoderma*, as well as those from environmental sequencing and other misidentified entries (though they do not state clearly how they found the latter data). They excluded sequences of short length (<350 bases), and sequences mistakenly labeled as *Ganoderma* (what criteria they use for this is not clear, but I would use a sequence similarity cutoff of ~85%). The resulting sequences were compared for similarity and sets of identical sequences were considered amplicon sequence variants (ASV) (so a group of identical sequences were treated like a single sequence) which were preferentially used for phylogenetic analyses – Maximum Likelihood (via RaxML) and Bayesian Inference (via MrBayes). 
+
+They differentiated species with thresholds for sequence similarity – that is, ITS sequence similarity above 97-98% are indicative of same-species, which is a widely-adopted threshold for species delimitation in Basidiomycota (Nilsson et al. 2008). However, each case was considered individually as this threshold wasn’t suitable for all cases. They also considered if the terminal subclade was well-supported, and whether any overlap existed between intraspecific genetic distance vs. respective interspecific values compared to the closest related taxon (so essentially, more closely related to each other than the next closest clade). 
```

Comment:
how was "each case (was) considered individually" in that paper? Is the sentence following it meant to expand on this point? if so, it wasn't clear to me.